gitlab/README.md

```diff
@@ -72,8 +70,6 @@ For containerized environments, see the [Autodiscovery Integration Templates][6]
 
```

Comment:
Should this containerized instance example be updated? https://github.com/DataDog/integrations-core/blob/master/gitlab/README.md#metric-collection-1